app/views/shared/_contact_panels.html.erb

```diff
@@ -28,9 +28,17 @@
         </div>
 
         <ul class="t-chat-opening-times contact-panel__list unstyled-list">
-          <% chat_opening_hours.open_periods.each do |period| %>
-            <li class="contact-panel__additional-info"><%= period.html_safe %></li>
+
+          <% unless alternate_hours_active? %>
+            <% chat_opening_hours.open_periods.each do |period| %>
+              <li class="contact-panel__additional-info"><%= period.html_safe %></li>
+            <% end %>
+          <% end %>
+
+          <% if display_migration_message? %>
```

Comment:
is about chat migration. Can we change to display_chat_migration_message? Then the subject is not implicit.